DOCSUP-14537: Document the log_queries_probability setting (#29099)
ClickHouse/ClickHouse: docs/en/operations/system-tables/query_log.md

2.  If an error occurred during query processing, two events with the `QueryStart` and `ExceptionWhileProcessing` types are created.
3.  If an error occurred before launching the query, a single event with the `ExceptionBeforeStart` type is created.

> You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to set the probability with which queries are registered in the `query_log` table.

Reviewer: ```suggestion
You can use the [log_queries_probability](../../operations/settings/settings.md#log-queries-probability) setting to reduce the number of queries, registered in the `query_log` table.
```
